### PR TITLE
ENH: uninstall --kill (just wipes it out) (Closes #876)

### LINKED ISSUE #876: uninstall --kill
I would like to have an option whenever I could just say -- kill (rmtree) the beast, without any checks/guards/etc.

### PR DESCRIPTION
**Pull request by yarikoptic**
quick feedback @bpoldrack and @mih would be appreciated!

### CODE REVIEW

File: datalad/distribution/uninstall.py [hunk 1]
```diff
@@ -171,7 +180,7 @@ def __call__(
             raise ValueError("`remove_history` flag, requires `remove_handles` flag")
 
         if not remove_data and not remove_handles:
-            raise ValueError("instructured to neither drop data, nor remove handles: cannot perform")
+            raise ValueError("instructed to neither drop data, nor remove handles: cannot perform")
```
Comment (bpoldrack) [inline datalad/distribution/uninstall.py]: This conflicts with your description of kill, reading "without consideration of being a dataset, a file, or a directory or any other option given to uninstall"

**Comment by bpoldrack** [comment]
Well, it says it's dangerous, so - may be. But: `rm -rf` is already there. Why do we need to proxy it, suggesting, that it somehow is a datalad operation?

**Comment by yarikoptic** [comment]
`rm -rf` is not good enough due to ro permissions of the annex load ;-)

**Comment by bpoldrack** [comment]
Well, good point.

**Comment by yarikoptic** [comment]
@mih wants it so here it comes ;)

**Pull request merged**